debian/rules

```diff
@@ -1,3 +1,20 @@
 #!/usr/bin/make -f
+
+DEB_BUILD_OPTIONS=nocheck
+
 %:
 	dh $@
+
+override_dh_auto_install:
```

Comment:
Terribly sorry for late response. I haven't seen email notification in my email.

I'm not sure I know how to do it with autotools. But my motivation was to make files tree different for deb and rpm.
For instance, we have different configs to place in result `etc` folder, depends on package type; rpm version has its specific files such as `statsite.tmpfiles.conf` and `statsite.initscript`.